packages/labs/ssr-client/package.json

```diff
@@ -18,10 +18,25 @@
   },
   "exports": {
     ".": {
+      "types": "./development.index.d.ts",
```

Comment:
Should this be `"./development/index.d.ts"`?

Reply:
Thanks for the find. Fixed.